UnityPy/export/Texture2DConverter.py

```diff
@@ -166,7 +165,7 @@ def parse_image_data(
     texture_format: Union[int, TextureFormat],
     version: tuple,
     platform: int,
-    platform_blob: bytes = None,
+    platform_blob: Union[bytes, None] = None,
```

Comment:
Please change the ``Union[foo, None]`` to ``Optional[foo]``, as that's what I've been using everywhere in UnityPy.

I think that Optional makes the implication more clear than a Union with None.